VideoLocker/src/androidTest/java/org/edx/mobile/test/feature/interactor/RegistrationScreenInteractor.java

```diff
@@ -27,7 +27,6 @@ public RegistrationScreenInteractor observeRegistrationScreen() {
         final CharSequence title = ResourceUtil.getFormattedString(app.getResources(),
                 R.string.register_title, "platform_name",
                 app.getInjector().getInstance(Config.class).getPlatformName());
-        onView(withId(R.id.activity_title)).check(matches(withText(title.toString())));
```

Comment:
Without this assertion, this method does nothing :(

Again, let's update this to check the title is correct.